interfaces/apparmor/spec.go

```diff
@@ -92,7 +92,12 @@ func (spec *Specification) AddSnippet(snippet string) {
 
 // AddUpdateNS adds a new apparmor snippet for the snap-update-ns program.
 func (spec *Specification) AddUpdateNS(snippet string) {
-	spec.updateNS = append(spec.updateNS, snippet)
+	spec.updateNS.Put(snippet)
+}
+
+// UpdateNSIndexOf returns the index of a previously added snippet.
+func (spec *Specification) UpdateNSIndexOf(snippet string) (idx int, ok bool) {
```

Comment:
Is is just for the tests? Can it go to exports?

Reply:
Yes and no :)

Reply:
Ok, can you clarify please? At the moment is used in tests only afaict?

Reply:
Yes but it is used from multiple packages.

Reply:
Ah, got it, thanks!